builtin/providers/aws/resource_aws_rds_cluster_instance.go

```diff
@@ -148,6 +154,10 @@ func resourceAwsRDSClusterInstanceCreate(d *schema.ResourceData, meta interface{
 		createOpts.MonitoringInterval = aws.Int64(int64(attr.(int)))
 	}
 
+	if attr, ok := d.GetOk("promotion_tier"); ok {
```

Comment:
This isn't needed - there is always a default value - so you can just add the parameter directly to the input